src/listeners/bush/appealListener.ts

```diff
@@ -0,0 +1,75 @@
+/* eslint-disable no-control-regex */
+import { BushListener, ModLog, type BushClientEvents } from '#lib';
+
+export default class AppealListener extends BushListener {
+	public constructor() {
+		super('appealListener', {
+			emitter: 'client',
+			event: 'messageCreate',
+			category: 'bush'
+		});
+	}
+
+	public override async exec(...[message]: BushClientEvents['messageCreate']): Promise<void> {
+		if (message.author.id !== '855446927688335370' || message.embeds.length < 1) return;
+		const userId = message.embeds[0].fields?.find?.((f) => f.name === 'What is your discord ID?')?.value;
+		if (!userId) return;
+		const thread = await message.startThread({
+			name: `${message.embeds[0].fields!.find((f) => f.name === 'What type of punishment are you appealing?')!.value} appeal`
+		});
+		const user = await this.client.users.fetch(userId).catch(() => null);
+		if (!user) {
+			await thread.send({
+				embeds: [
+					this.client.util
+						.embed()
+						.setTitle(
+							`${message.embeds[0].fields!.find((f) => f.name === 'What type of punishment are you appealing?')!.value} appeal`
+						)
+						.addField({
```

Comment:
```suggestion
						.addFields({
```
discord.js removed addField